app/views/shared/_facility_complexity_modal.html.erb

```diff
@@ -1,6 +1,11 @@
-<a href="#facility-complexity-modal" aria-controls="facility-complexity-modal" data-open-modal class="fas fa-question-circle cursor-pointer">
+<div href="#facility-complexity-modal" class="facility-complexity-modal cursor-pointer" aria-controls="facility-complexity-modal" data-open-modal class="cursor-pointer">
```

Comment:
We should change any `<div></div>` element that has an `href` attr to a `<a></a>` element.